tools/topology/topology2/cavs/cavs-passthrough-hda.conf

```diff
@@ -16,7 +16,7 @@ Object.Dai {
 			stream_name 'Analog Playback and Capture'
 			period_sink_count 0
 			period_source_count 2
-			format s32le
+			format s16le
```

Comment:
?? Why would we restrict the format to 16bits?

Reply:
I'm going to drop the first patch and replace it to remove the format entirely @plbossart. It is never really used in the kernel or the fw